src/hotspot/share/opto/superwordVTransformBuilder.cpp

```diff
@@ -175,10 +177,9 @@ VTransformVectorNode* SuperWordVTransformBuilder::make_vector_vtnode_for_pack(co
   return vtn;
 }
 
-void SuperWordVTransformBuilder::set_req_with_scalar(Node* n, VTransformNode* vtn, VectorSet& vtn_dependencies, const int index) {
+void SuperWordVTransformBuilder::set_req_with_scalar(Node* n, VTransformNode* vtn, const int index) {
   VTransformNode* req = get_vtnode_or_wrap_as_input_scalar(n->in(index));
   vtn->set_req(index, req);
-  vtn_dependencies.set(req->_idx);
```

Comment:
Not useful any more. We used to track all dependencies this way, but we really only need to track the memory dependencies.